docs/document-parts.md

```diff
@@ -75,70 +152,63 @@ parts:
-If you are rendering your project in other places, it can be helpful to leave these sections directly in the document.
-Complete this using a header as usual:
-
-```
-# Abstract
-
-This is my abstract!
-```
-
-Note that frontmatter parts and explicitly tagged cells/blocks will take precedence over this method. Themes may choose to only pick up a subset of implicit parts, for example, only an `Abstract` and not `Summary` as summary section can be used in other contexts.
-
-(parts:blocks)=
-
-## In a Jupyter Notebook cells and blocks
+(parts:site)=
 
-When using a Jupyter Notebook, you can add a `tag` to the cell with the part name, if multiple cells share that tag, they will be extracted and merged. This can also be represented in a [block](./blocks.md):
+## Add parts to your website
 
-```markdown
-+++ { "part": "abstract" }
+[Website themes](./website-templates.md) have their own configuration for parts because they often have extra user interface elements that are not part of a standard "document" structure. These are theme-dependent. For example, the [default myst themes](#default-web-themes) support [a website footer](#navigation:footer).
 
-This is my abstract block.
+Specify the content of a site `part` in the `site.parts` key of the `myst.yml` configuration file:
 
-+++
+```{code} yaml
+:filename: myst.yml
+version: 1
+site:
+  template: ...
+  parts:
+    footer: |
+      (c) MyST Markdown
+  ...
 ```
 
-(parts:project)=
-
-## Parts in `myst.yml` Project configuration
-
-You may also specify `parts` in the project configuration of your `myst.yml` file. These are defined exactly the same as [`parts` defined in page frontmatter](#parts:frontmatter).
+Alternatively, you may specify a path to a file that contains the part content:
 
-```yaml
+```{code} yaml
+:filename: myst.yml
 version: 1
-project:
-  abstract:  |
-    This is a multi-line
-    abstract, with _markdown_!
+site:
+  template: ...
   parts:
-    special_part: |
-      This is another _special_ part!
+    footer: parts/myfooter.md
+  ...
 ```
 
-Project-level `parts` are useful, for example, if you have an abstract, acknowledgments, or other part that applies to your entire project and doesn't make sense attached to an individual page.
+Content in parts will generally be parsed similarly to other MyST content (though some functionality like code execution will not work).
 
-```{caution}
-Project-level `parts` are a new feature and may not yet be respected by your chosen MyST template or export format. If the project `part` is not behaving as you expect, try moving it to page frontmatter for now.
-```
+### Share the same part across multiple sites
 
-(parts:site)=
+If you wish to share the same part across multiple sites, use the [`extends:` key to compose multiple configuration files](#composing-myst-yml). This lets you define the part in one location, and re-use it in several others. It is useful if you want to standardize website components like a footer across many websites.
```

Comment:
How does composing configuration files help here? E.g., is it easier to pull in the config that pulls in the part? Or can a person pull the part in directly? (Maybe not possible?)

Reply:
But wouldn't you just want to point to that *part* in your current site config?

Reply:
I don't think that is possible if the part isn't local, at least until this enhancement is made:

- https://github.com/jupyter-book/mystmd/issues/2127